ios/Capacitor/Capacitor/CAPInstanceDescriptor.swift

```diff
@@ -134,6 +134,9 @@ internal extension InstanceDescriptor {
             if let handleNotifications = config[keyPath: "ios.handleApplicationNotifications"] as? Bool {
                 handleApplicationNotifications = handleNotifications
             }
+            if let webContentsDebuggingEnabled = config[keyPath: "ios.webContentsDebuggingEnabled"] as? Bool {
+                isWebDebuggable = webContentsDebuggingEnabled
+            }
```

Comment:
```suggestion
            } else {
                #if DEBUG
                    isWebDebuggable = true
                #endif
            }
```
set the default to true for debug builds if `webContentsDebuggingEnabled` is not set